refactor: update navbar background blur

Inushin opened the pull request:
Update the navbar background blur to improve readability.

Inushin committed:
refactor: update navbar background blur

Makosai commented:
This adds a weird look to the navbar.
![image](https://github.com/user-attachments/assets/88d38439-4861-4889-b69b-2b1e46031f02)

An alternative we can do is something that doesn't ruin the aesthetics by just using a shadow on top of the image to improve the image. The challenge is to make it so the image doesn't appear overly dark everywhere. So the shadow has to be crafted intentionally only for the text. That has mostly been done already but if you feel like it needs more we can discuss that here. :)

Inushin commented:
Sorry, I should have mentioned that I saw this on mobile. If that is what you were looking for, my bad. Then, it might be interesting to hide the hamburger menu that opens the navbar, and remove the Reia logo if it's already displayed in the navbar.

![Screenshot_2025-03-29-11-17-36-206_com.brave.browser-edit.jpg](https://github.com/user-attachments/assets/5c6d4985-303f-43c9-88a6-880ca99f0c3d)

Makosai commented:
Okay, now that I did some work on the game / netcode, I'm going to jump back into the frontend. While I was working on the other projects for Reia, I got a better understanding on how to use Paraglide as well. So I'll be opening up an issue to implement translations throughout the entire app. :) The hard part might be using it in MDsveX.

I should have some updates on this PR later today!

Makosai committed:
Merge branch 'main' into pr/60

Makosai committed:
Fix the blur for the mobile sidebar.

The issue here was that when the app is built, it prefers -webkit-backdrop-filter over backdrop-filter. But -webkit-backdrop-filter doesn't exist. So it got rid of the one that existed and kept the one that didn't. This has been resolved.

Makosai commented:
The issue here was that when the app is built, it prefers `-webkit-backdrop-filter` over backdrop-filter. But `-webkit-backdrop-filter` doesn't exist. So it got rid of the one that existed and kept the one that didn't. This has been resolved. You can look at `colors.css` to get a feel for where the issue was.

Makosai committed:
Revert the blur change for desktop.
